docs/vgclog.md

```diff
@@ -0,0 +1,268 @@
+
+  <tr>
+    <th scope="col">Example log line number</th>
+    <th align="center" scope="col">`gencon` global GC cycle status recorded in log</th>
+    <th align="center"scope="col">`gencon` partial GC cycle status recorded in log</th>
+  </tr>
+  <tr>
+    <td>1-87</td>
+    <td colspan="2" align="center">Initialization section of the logs</th>
+  </tr>
+  <tr>
+    <td scope="row"> 87-51676</td>
+    <td align="center"> - </td>
+    <td align="center"> series of partial scavenge cycles start and finish </td>
+  </tr>
+
+  <tr>
+    <td scope="row"> 51677</td>
+    <td align="center"> global cycle's trigger and target logged</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51680</td>
+    <td align="center"> STW pause starts</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51683</td>
+    <td align="center"> global cycle starts</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51684</td>
+    <td align="center"> STW pause ends</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">518685</td>
+    <td align="center"> blank line in logs. (Concurrent increment runs) </td>
+    <td align="center">-</td>
+  </tr>
+  <tr>
+    <td scope="row">51686</td>
+    <td align="center"> (concurrent increment paused)</td>
+    <td align="center">STW pause starts</td>
+  </tr>
+    <tr>
+    <td scope="row">51690</td>
+    <td align="center"> (concurrent increment paused)</td>
+    <td align="center">partial scavenge cycle starts</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51691</td>
+    <td align="center"> (concurrent increment paused)</td>
+    <td align="center"> partial scavenge increment runs</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51730</td>
+    <td align="center"> (concurrent increment paused)</td>
+    <td align="center">partial cycle ends</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51733</td>
+    <td align="center"> (concurrent increment resumes)</td>
+    <td align="center">STW pause ends</td>
+  </tr>
+
+   <tr>
+    <td scope="row">51734</td>
+    <td align="center"> blank line in logs. (Concurrent increment resumes)</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51735</td>
+    <td align="center"> STW pause starts</td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51741</td>
+    <td align="center"> final global increment logged </td>
+    <td align="center">-</td>
+  </tr>
+  <tr>
+    <td scope="row">51793</td>
+    <td align="center"> global cycle ends </td>
+    <td align="center">-</td>
+  </tr>
+
+  <tr>
+    <td scope="row">51795</td>
+    <td align="center"> STW pause ends</td>
+    <td align="center">-</td>
+  </tr>
+</table>
+
+**Note:** Zero, one, or multiple gc cycles might run between the start and end of a `gencon` global GC cycle.
```

Comment:
Please use font awesome icons with notes and restrictions. See contributing guidelines for markup.